openlattice.yaml

```diff
@@ -2766,6 +2767,56 @@ paths:
                 items:
                   $ref: "#/components/entityLinkingFeatures"
 
+  ################
+  ## ADMIN APIs ##
+  ################
+  
+  /indexer/index/reindex:
+    get:
+      summary: Retrieve the current state of reindexing jobs
+      operationId: getIndexingState
+      tags:
+        - admin
+      security:
+        - openlattice_auth: []
+        - http_auth: []
+      responses:
+        "200":
+          description: Success
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/indexingState"
+    post:
+      summary: Merge job descriptions for performing a partial or full reindex of provided entity sets.
+      operationId: reindex
+      tags:
+        - admin
+      security:
+        - openlattice_auth: []
+        - http_auth: []
+      requestBody:
+        required: true
+        description: A map of entity set ids to entity key ids that determines what will be reindexed. If no entity key ids are provided then all entities in an entity set are reindexed. If no entity set ids are provided then all entity sets are scheduled for reindexing.
+        content:
+          application/json:
+            schema:
+              type: object
+              additionalProperties:
+                type: array
+                items:
+                  type: string
+                  format: uuid
+      responses:
+        "200":
+          description: Success
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/indexingState"
+    
```

Comment:
I'm not sure, if this is intentional or not, but shouldn't be the `PUT` and `DELETE` requests also be described here?

Reply:
Yeah, I usually add paths as we need them, but they're added now :) .